From 25700873e499a7611158f6cd50511d028d37adda Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 26 Dec 2024 11:40:02 +0100
Subject: [PATCH 46/N] fixup! Avoid sharing cygheaps across Cygwin versions

There was a spurious double-quote in the error message that is shown
when the commit could not be determined.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/configure.ac b/winsup/configure.ac
index 0fc607a..4dd5ccb 100644
--- a/winsup/configure.ac
+++ b/winsup/configure.ac
@@ -74,7 +74,7 @@ yes|auto)
         MSYS2_RUNTIME_COMMIT_SHORT="$(expr "$MSYS2_RUNTIME_COMMIT" : '\(.\{,8\}\)')"
         MSYS2_RUNTIME_COMMIT_HEX="0x${MSYS2_RUNTIME_COMMIT_SHORT}ul"
     else
-        AC_MSG_WARN([Could not determine msys2-runtime commit"])
+        AC_MSG_WARN([Could not determine msys2-runtime commit])
         MSYS2_RUNTIME_COMMIT=
         MSYS2_RUNTIME_COMMIT_SHORT=
         MSYS2_RUNTIME_COMMIT_HEX=0
